turingschool-examples/romcom
# Save cover view

##What is the change?
FEATURE: When you click on View Saved Covers button:
- The Saved Covers view is visible (there is nothing on it right now)
- The home button becomes visible
- The View Saved Covers button becomes hidden
- The Create Random Cover button becomes hidden

##Where should the reviewer start?
Line 59 - the function viewSavedCovers which references the variables in lines 14/15 and the event listener in line 29.

##How should this be tested?
Open index.html, click on the View Saved Covers button and the buttons (described above) should be added/hidden, the home view should be hidden and the view saved cover page (blank) should come into view.
